src/API/AdvertiserConnect.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Handle Pinterest Advertiser Connect
+ *
+ * @package     Pinterest_For_Woocommerce/API
+ * @version     1.0.0
+ */
+
+namespace Automattic\WooCommerce\Pinterest\API;
+
+use \WP_REST_Server;
+use \WP_REST_Request;
```

Comment:
Two things: 

* We should import the `WP_Error` class, because it is referenced in a code comment and as a possible return value (line 74)
* When importing, the leading `\` character is unnecessary and should be skipped.

```suggestion
use WP_Error;
use WP_REST_Server;
use WP_REST_Request;
```